### ISSUE TITLE
Copy Module Fails with relatively large files

### ISSUE THREAD

**Issue by ansibot** [ansible/ansible#29525]
From @abassanini on 2016-11-03T17:15:49Z
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
Copy module

##### ANSIBLE VERSION
```
ansible 2.3.0 (devel 32a7b4ce71) last updated 2016/11/03 11:04:12 (GMT -500)
  lib/ansible/modules/core: (detached HEAD 7cc4d3fe04) last updated 2016/11/03 11:04:33 (GMT -500)
  lib/ansible/modules/extras: (detached HEAD e4bc618956) last updated 2016/11/03 11:04:54 (GMT -500)
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
```
$ cat /etc/ansible/hosts
local	ansible_host=127.0.0.1
test_host	ansible_host=192.168.56.50

$ egrep -v "^#|^$" /etc/ansible/ansible.cfg
[defaults]
log_path = /var/log/ansible.log
[privilege_escalation]
[paramiko_connection]
[ssh_connection]
[accelerate]
[selinux]
[colors]
```
##### OS / ENVIRONMENT
```
$ hostnamectl
   Static hostname: ansible
         Icon name: computer-vm
           Chassis: vm
           Boot ID: 44bef02e34ee4cad9ddf55df52cb03c5
  Operating System: Ubuntu 14.04.5 LTS
            Kernel: Linux 3.13.0-100-generic
      Architecture: x86_64
```

##### SUMMARY
When trying to copy a large file (jdk installer 195Mb) the copy module fails, but works perfectly with small files (conf files for example)

##### STEPS TO REPRODUCE
Run an ad-hoc command with the large file

```
ansible -vvv test_host -s -m copy -a 'src=/vagrant/jdk-8u111-windows-x64.exe dest=/var/www/html/ owner=www-data group=www-data mode=0644'

```

##### EXPECTED RESULTS
Expected a successfully file copy

##### ACTUAL RESULTS
It fails with MemoryError

```
Using /etc/ansible/ansible.cfg as config file
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/home/vagrant/ansible/lib/ansible/executor/task_executor.py", line 119, in run
    res = self._execute()
  File "/home/vagrant/ansible/lib/ansible/executor/task_executor.py", line 490, in _execute
    result = self._handler.run(task_vars=variables)
  File "/home/vagrant/ansible/lib/ansible/plugins/action/copy.py", line 157, in run
    source_full = self._loader.get_real_file(source_full)
  File "/home/vagrant/ansible/lib/ansible/parsing/dataloader.py", line 402, in get_real_file
    if is_encrypted_file(f):
  File "/home/vagrant/ansible/lib/ansible/parsing/vault/__init__.py", line 152, in is_encrypted_file
    b_vaulttext = to_bytes(to_text(vaulttext, encoding='ascii', errors='strict'), encoding='ascii', errors='strict')
  File "/home/vagrant/ansible/lib/ansible/module_utils/_text.py", line 177, in to_text
    return obj.decode(encoding, errors)
MemoryError

test_host | FAILED! => {
    "failed": true,
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}
```
Copied from original issue: ansible/ansible-modules-core#5478

**Comment by ansibot** [comment]
From @ansibot on 2016-11-03T17:15:49Z

@abassanini, we need more information to resolve this issue. For that reason, we put this Issue into the 'needs_info' state.

Here are the required items we could not find in your description:
- issue type

When you have filled in the missing data, we will notify the module maintainer for further action.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @abassanini on 2016-11-03T17:15:49Z

Info added

**Comment by ansibot** [comment]
From @ansibot on 2016-11-03T17:15:49Z

@ansible, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
From @abassanini on 2016-11-03T17:15:49Z

ISSUE TYPE
Bug Report

## 

Saludos,
Aldo

**Comment by ansibot** [comment]
@abassanini This issue is waiting for your response. Please respond or the issue will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@abassanini You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
